data-raw/un-data.R

```diff
@@ -2,46 +2,60 @@ library(dplyr)
 library(readxl)
 library(countrycode)
 library(tidyr)
-library(devtools)
+library(forcats)
+library(usethis)
 
 vlevels <- c("yes", "abstain", "no")
 
-load("data-raw/UNVotesPublished.RData")
-un_votes <- x %>%
-  tbl_df() %>%
+load("data-raw/UNVotes2021.RData")
+
+un_votes <- completeVotes %>%
   filter(vote <= 3) %>%
-  mutate(country = countrycode(ccode, "cown", "country.name"),
-         country_code = countrycode(ccode, "cown", "iso2c")) %>%
+  mutate(
+    country = countrycode(ccode, "cown", "country.name"),
+    country_code = countrycode(ccode, "cown", "iso2c"),
+    # Match based on old version of data from unvotes package
+    country_code = case_when(
+      country == "Czechoslovakia" ~ "CS",
+      country == "Yugoslavia" ~ "YU",
+      country == "German Democratic Republic" ~ "DD",
+      country == "Yemen People's Republic" ~ "YD",
+      TRUE ~ country_code
+    ),
+    country = if_else(Countryname == "German Federal Republic", "Federal Republic of Germany", country)
```

Comment:
One bug here: looks like this needs to be `!is.na(Countryname) & Countryname == "German Federal Republic"`, otherwise a few thousand countries end up NA because the countryname was NA 🙂